### DISCUSSION TITLE
Is there a way?

### DISCUSSION THREAD

**Discussion by ghost** [scandum/tintin#105] [General]
What's the good way...? ? Screen puts tintin++ in the background, and then executes the instructions inside tintin++ outside??

**Comment by prool** [comment]
I did exactly that in my MUD statistic system. For example https://github.com/prool/prool-utilities/blob/master/russian-mud-statistics/adan.sh

prooltin is my simply fork of tintin++. I'm make new command #watchdogtimer for avoid infinite loop in scripts

**Comment by scandum** [comment]
There's `#daemon` to run tintin in the background.
